website/cue/reference/remap/functions/parse_influxdb.cue

```diff
@@ -0,0 +1,108 @@
+package metadata
+
+remap: functions: parse_influxdb: {
+	category: "Parse"
+	description: """
+		Parses the `value` as an [InfluxDB line protocol](https://docs.influxdata.com/influxdb/cloud/reference/syntax/line-protocol/)
+		string, producing a list of Vector-compatible metrics".
+		"""
+	notices: [
+	   """
+		    This function will return a log event with the shape of a Vector-compatible metric, but not a metric event itself.
+		    You will likely want to pipe the output of this function through a `log_to_metric` transform with the option `all_metrics`
+			set to `true` to convert the metric-shaped log events to metric events so _real_ metrics are produced.
+		"""
+			The only metric type that is produced is a `gauge`. Each metric name is prefixed with the `measurement` field, followed
+			by an underscore (`_`), and then the `field key` field.
+			""",
+	    """
+			`string` is the only type that is not supported as a field value,
+			due to limitations of Vector's metric model.
+			""",
+	]
+	arguments: [
+		{
+			name:        "value"
+			description: "The string representation of the InfluxDB line protocol to parse."
+			required:    true
+			type:        ["string"]
+		},
+	]
+	internal_failure_reasons: [
+		"`value` is not a valid InfluxDB line protocol string.",
+		"field set contains a field value of type `string`.",
```

Comment:
Added two more failure reasons, related to https://github.com/vectordotdev/vrl/blob/b0daea0245de03ecae01622417f17093c8736594/src/stdlib/parse_influxdb.rs#L77